### PR TITLE
docs: add a quick start page to the readme and docs

### PR DESCRIPTION
**Pull request by MthwRobinson**
### Summary

Adds a quick start page to the readme and docs to make it easier for users to get up and running with the package faster.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,11 +49,48 @@ about. Bricks in the library fall into three categories:
...
+    - `poppler-utils`
+    - `tesseract-ocr`
+    - `libreoffice`
+- Run the following to install NLTK dependencies. `unstructured` will handle this automatically
+  soon.
+  - `python -c "import nltk; nltk.download('punkt')"`
+  - `python -c "import nltk; nltk.download('averaged_perceptron_tagger')"`
+- If you are parsing PDFs, run the following to install the `detectron2` model, which
+  `unstructured` uses for layout detection:
+    - `pip install "detectron2@git+https://github.com/facebookresearch/detectron2.git@v0.6#egg=detectron2"`
```
Comment (qued) [inline README.md:71]: Not specific to this PR, but this says to install 0.6, while in `unstructured-inference` the readme says to install 0.4, while the `Makefile` installs some specific commit that's not either version. We need to get everything on the same page.

File: README.md [hunk 2]
```diff
@@ -49,11 +49,48 @@ about. Bricks in the library fall into three categories:
...
+
+At this point, you shold be able to run the following code:
```
Comment (qued) [inline README.md:73]:
```suggestion
At this point, you should be able to run the following code:
```

File: README.md [hunk 3]
```diff
@@ -49,11 +49,48 @@ about. Bricks in the library fall into three categories:
...
   and data labeling.
 <br></br>
-## :eight_pointed_black_star: Installation
+## :eight_pointed_black_star: Quick Start
+
+Use the following instructions to get up and running with `unstructured` and test your
+installation.
+
+- Install the Python SDK with `pip install unstructured[local-inference]`
+		- If you do not need to process PDFs or images, you can run `pip install unstructured`
```
Comment (qued) [inline README.md:58]: A mix of tabs (like here) and spaces for indentation.

File: docs/source/installing.rst [hunk 4]
```diff
@@ -1,10 +1,43 @@
...
+
+  elements = partition(filename="example-docs/fake-email.eml")
+
+And if you installed with `local-inference`, you should be able to run this as well:
+
+.. code:: python
+
+  from unstructured.partition.auto import partition
+
+  elements = partition("example-docs/layout-parser-paper.pdf")
```
Comment (qued) [inline docs/source/installing.rst:40]: Same comments as other file.

**Review by qued — APPROVED** [review]
LGTM other than nits.

File: README.md (hunk 1 above)
Comment (MthwRobinson) [reply README.md:71]: Yeah, 🤞 we can just get rid of detectron soon

**Comment by MthwRobinson** [comment]
Added #241 for syncing the `detectron2` docs

**Pull request merged**